netmiko/__init__.py

```diff
@@ -23,7 +23,7 @@
 NetmikoAuthError = NetMikoAuthenticationException
 Netmiko = ConnectHandler
 
-__version__ = "2.3.3"
+__version__ = "2.3.4"
```

Comment:
Is there a reason you bumped the version? That looks like a job reserved for the maintainers.

Reply:
I did not know about that. I have few expierence with Github.

Thank you for telling me that!